tools/maven-plugin/src/main/java/io/smallrye/openapi/mavenplugin/MavenDependencyIndexCreator.java

```diff
@@ -59,6 +59,7 @@ public MavenDependencyIndexCreator() {
         ignoredArtifacts.add("com.thoughtworks.xstream:xstream");
         ignoredArtifacts.add("com.github.javaparser:javaparser-core");
         ignoredArtifacts.add("org.jboss:jandex");
+        ignoredArtifacts.add("org.jboss.resteasy:resteasy-core");
```

Comment:
In the future, it might be better to remove the whole static artifact blacklist from the maven plugin.
We should do the same as quarkus does for building the CombinedIndexBuildItem.
* search in each dependency for Marker files (like META-INF/jandex.idx, META-INF/beans.xml)
* Only include matching dependencies in the index

At least that would be consistent, and (imo) the expected behaviour.

@phillip-kruger WDYT?

Reply:
Sounds good to me! Thanks